### PR TITLE
Feature/molo 568 page scroll update

### PR DESCRIPTION
**Pull request by projectmushroom**
- vertical scrolling touchpad bug fixed
- scroller added on other admin views (fixes comment list refresh error)
- overlapping edit/delete controls fix

### CODE REVIEW

**Comment by Mitso** [comment]
@projectmushroom looks great!!
🍰 🍰 🍰 🍰 🍰 🍰 

Couple of issues I can picked up:
Filter dates popup are behind container 
screen shot 2017-11-21 at 10 38 31 am

Filter scrolls looks different in color from other Admin xy scroll.
The trackpad scroll direction is opposite the other scroll & as per setting
screen shot 2017-11-23 at 9 54 25 am

**Comment by Mitso** [comment]
Date filter is updated.

screen shot 2017-11-23 at 10 57 19 am

Nevermind the second scroll color issue. It's On Hover, sorry

File: molo/core/styles/wagtail-admin/modules/_m-filter.scss [hunk 1]
```diff
@@ -64,4 +68,10 @@ header {
       padding: 0 5px 0 5px;
     }
   }
+
+  body.admin-view  {
+      .changelist-filter {
+        position: relative;
```
Comment (projectmushroom) [inline molo/core/styles/wagtail-admin/modules/_m-filter.scss]: @Mitso The Fly out doesn't like the relative position
screen shot 2017-11-23 at 12 11 07

File: molo/core/templates/admin/date_range_filter.html [hunk 2]
```diff
@@ -3,15 +3,8 @@
...
-    $('#id_drf__submission_date__gte').datetimepicker({
-      timepicker: false,
-      format: 'Y-m-d',
-      i18n: {
-          lang: window.dateTimePickerTranslations
-      },
-      lang: 'lang'
-    });
-    $('#id_drf__submission_date__lte').datetimepicker({
+    console.log("hey");
```
Comment (nathanbegbie) [inline molo/core/templates/admin/date_range_filter.html]: @projectmushroom can we remove this in a future PR please? 😆

Comment (projectmushroom) [reply molo/core/templates/admin/date_range_filter.html]: haha, oh snap thought I cleaned it!

**Pull request merged**